metadata-io/src/main/java/com/linkedin/metadata/search/SearchService.java

```diff
@@ -20,6 +20,13 @@
    */
   void clear();
 
+  /**
+   * Get the number of documents corresponding to the entity
+   *
+   * @param entityName name of the entity
+   */
+  long docCount(@Nonnull String entityName);
```

Comment:
How does this hold up given Timeseries aspects? Can we call this getSnapshotDocCount or something of the type to illustrate this divergence?

Reply:
Timeseries service doesn't implement SearchService. This is specific to search backend (which is only used by versioned aspects)

Reply:
Oh okay got it